src/os_posix/os_alloc.c

```diff
@@ -9,6 +9,27 @@
 #include "wt_internal.h"
 
 /*
+ * On systems with poor default allocators for allocations greater than 16 KB,
+ * we provide an option to use TCMalloc explicitly.
+ * This is important on Windows which does not have a builtin mechanism
+ * to replace C run-time memory management functions with alternatives.
+ */
+#ifdef HAVE_LIBTCMALLOC
+#include <gperftools/tcmalloc.h>
+
+/*
+ * Define HAVE_POSIX_MEMALIGN explicitly since TCMalloc supports it
+ */
+#define	HAVE_POSIX_MEMALIGN 1
```

Comment:
@keithbostic Thanks for that fix. I tend to focus on the Windows port so I missed the buffer alignment.